tests/e2e/cucumber/features/smoke/fullTextSearch.ocis.feature

```diff
@@ -0,0 +1,89 @@
+Feature: Search
+  As a user
+  I want to do full text search
+  So that I can find the files with the content I am looking for
+
+  Background:
+    Given "Admin" creates following users using API
+      | id    |
+      | Alice |
+      | Brian |
+    And "Admin" sets the default folder for received shares to "Shares"
+    And "Admin" disables share auto accepting
+    And "Admin" assigns following roles to the users using API
+      | id    | role        |
+      | Brian | Space Admin |
+    And "Alice" logs in
+    And "Alice" creates the following folder in personal space using API
+      | name          |
+      | folderToShare |
+    And "Alice" uploads the following local file into personal space using API
+      | localFile                   | to                            |
+      | filesForUpload/textfile.txt | folderToShare/fileToShare.txt |
+    And "Alice" shares the following resource using API
+      | resource      | recipient | type | role     |
+      | folderToShare | Brian     | user | Can edit |
```

Comment:
Would it make sense to add this step here? I don't see Alice user used anywhere else
```suggestion
    And "Alice" shares the following resource using API
      | resource      | recipient | type | role     |
      | folderToShare | Brian     | user | Can edit |
      And "Alice" logs out
```

Reply:
The share is created to search item in accepted share space of brian.

Reply:
I meant to say `Alice` logs in but I don't see the logging out step so maybe we can add it right before `Brian` logs in? I didn't see `Alice` user used anywhere after that step